Source/Mosa.Plug.Korlib/System.IO/FilePlug.cs

```diff
@@ -12,7 +13,7 @@ public static class FilePlug
 		[Plug("System.IO.File::ReadAllBytes")]
 		internal static byte[] ReadAllBytes(string path)
 		{
-			var fat = HAL.GetCurrentFileSystem();
+			var fat = KernelImpl.GetCurrentFileSystem();
```

Comment:
This is a major hack and not a solution. Let's find a more elegant way of doing this.

Reply:
A separate project isn't elegant? Then what is?

Reply:
See discord discussion.